js/index.js

```diff
@@ -1,12 +1,13 @@
-const btnToggle = document.querySelector('#btn-toggle');
-const sidebarList = document.querySelector('#sidebar-nav');
+const btnToggle = document.querySelector("#btn-toggle");
```

Comment:
i would suggest if youre using ID keep less generic names, otherwise looks nice

Reply:
Fixing this in next PR